### ISSUE TITLE
header

### ISSUE THREAD

**Issue by texrg** [michael105/et#1]
make

gcc -o et minilib.c -I . et.c
In file included from minilib.c:1:
minilib.h:59:10: fatal error: minilib/include/minilib_header.h: No such file or directory
 #include "minilib/include/minilib_header.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from et.c:47:
minilib.h:59:10: fatal error: minilib/include/minilib_header.h: No such file or directory
 #include "minilib/include/minilib_header.h"

**Comment by michael105** [comment]
Thanks a lot.
That's been a dependency onto an older version of "minilib".
I now included a compiled version of the minilib, minilib.h.
With the consequence, since I didn't check minilib on other architectures yet,
this might work only on amd64 now.
At least, the problem with the missing header file is fixed for sure ;)
And I had to fix another problem, I didn't pinpoint the real source,
but (again) the optimizing of gcc did seem to cut some lines.
